state/payloads_test.go

```diff
@@ -73,28 +73,12 @@ func (s *PayloadsSuite) TestTrackInvalidPayload(c *gc.C) {
 }
 
 func (s *PayloadsSuite) TestTrackInvalidUnit(c *gc.C) {
-
-	// Note: this is STUPID, but none of the unit-specific contexts
```

Comment:
this still holds, AFAICS? why drop these comments? let's fix the code first, because the issues weren't obvious to me at least, without having looked closely